packages/bitcore-client/bin/wallet-sign

```diff
@@ -1,30 +1,64 @@
 #!/usr/bin/env node
+'use strict';
 
 const program = require('../ts_build/program');
 const { Wallet } = require('../ts_build/wallet');
 const promptly = require('promptly');
+const fs = require('fs');
+const { StreamUtil } = require('../ts_build/stream-util');
+const { Encryption } = require('../ts_build/encryption');
 
+/**
+ * USAGE -
+ * Pass in a raw transaction or a jsonl file of rawTransactions
+ * Pass in the bitcore wallet name or a jsonl file of encrypted bitcoin core keys or
+ * Pass in the utxos if you want to use specific utxos
+ */
 program
   .version(require('../package.json').version)
-  .option('--name <name>', 'REQUIRED - Wallet Name')
+  .option('--name <name>', 'REQUIRED - Bitcore Wallet Name')
   .option('--path [path]', 'optional - Custom wallet storage path')
-  .option('--tx <tx>', 'REQUIRED - The transaction to sign')
+  .option('--tx [tx]', 'The raw transaction to sign')
+  .option('--txfile [txfile]', 'path to a jsonl file with rawTransaction and utxos')
+  .option('--utxos', 'Pass in utxos to use to sign transaction')
+  .option('--keys [keys]', 'jsonl file of {privKey, pubKey, address} pairs')
   .parse(process.argv);
 
 const main = async () => {
-  const { name, path, tx } = program;
+  const { name, path, tx, txfile, keys, utxos } = program;
   let wallet;
+  let decryptedKeys;
   try {
     const password = await promptly.password('Wallet Password:');
     wallet = await Wallet.loadWallet({ name, path });
     wallet = await wallet.unlock(password);
-    let signedTx = await wallet.signTx({ tx });
-
-    console.log(
-      JSON.stringify({
-        signedTx: signedTx.uncheckedSerialize()
-      })
-    );
+    if (keys) {
+      let contents = fs.readFileSync(keys).toString();
+      decryptedKeys = contents
+        .split('\n')
+        .filter(line => line !== '')
+        .map(line => JSON.parse(line));
+      try {
+        const bitcoinCorePassword = await promptly.password('Enter Bitcoin Core Password:');
+        let decryptedParams = Encryption.bitcoinCoreDecrypt(decryptedKeys, bitcoinCorePassword);
+        decryptedKeys = decryptedParams.jsonlDecrypted;
+      } catch (e) {
+        console.error(e);
+      }
+    }
+    if (tx) {
+      let signedTx = await wallet.signTx({ tx, keys: decryptedKeys, utxos });
+      console.log(JSON.stringify(signedTx));
+    }
+    if (txfile) {
+      const readStream = fs.createReadStream(txfile, {flags: 'r', encoding: 'utf8'});
+      const writeStream = fs.createWriteStream('signedTx.jsonl');
```

Comment:
I'd rather this hit the console than output to a particular file. We can pipe the console to a file if we want